templates/deployment.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "torqbit.fullname" . }}
+  labels:
+    app: {{ include "torqbit.name" . }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ include "torqbit.name" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ include "torqbit.name" . }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - containerPort: 3000
+          env:
+            - name: DATABASE_URL
+              value: "mysql://kali:kali@mysql-service.default.svc.cluster.local:3306/kali"
```

Comment:
Use secrets to inject environment variables like - `DATABASE_URL`, `GOOGLE_ID` and so on..